data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+	
+	on visit
+		dialog "You have landed on <planet>, but you have not located the Star Queen! Find it somewhere in the Far North."
+	
+	to complete
+		has "found star queen"
+	
+	on complete
+		payment 100000
+
+mission "Bactrian Unlock [0]: Prime"
+	landing
+	source Prime
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`The Star Queen is not in this system, so you attempt to find somebody who may have some information. You eventually find a merchant who claims to have been here when the Star Queen was and that a large pirate fleet jumped in and took the Star Queen hostage before jumping north to Rigel.`
+				decline
+
+mission "Bactrian Unlock [0]: Farpoint"
+	landing
+	source Farpoint
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`After speaking with a Navy Officer for a few minutes, you find out that the Star Queen came through here over a week ago, escorted by a few large pirate ships. They did not want to risk the lives of those on the Star Queen by attacking the pirates, so they let the ship go and it continued north from here.`
+			`	There have been preparations for a rescue mission, but the fleet that originally captured the Star Queen had also destroyed a number of Navy capital ships, so they have been waiting for supplies before going out on the potentially dangerous mission.`
+			`	The Navy officer ends by hinting that the Star Queen is likely to be in Alnilam or Arneb, as those are the farthest places from anyone willing to try and rescue the ship.`
+				decline
+
+mission "Bactrian Unlock [0]: Pirates"
+	invisible
+	landing
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [0]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Leviathan (Laser)" "His Right Hand"
+		ship "Leviathan (Laser)" "His Left Hand"
+
+mission "Bactrian Unlock [1]"
+	name "Secure the Scientists"
+	description "Now that you have located the Scientists and their Star Queen, return to <destination> with the Deep escorts and retrieve it."
+	landing
+	destination Haven
+	stopover Memory
+	stopover Farpoint
+	clearance
+	infiltrating
+	to offer
+		has "Bactrian Unlock [0]: done"
+	on offer
+		conversation
+			`You return to <planet> to find the man that you met earlier in the spaceport along with a large group of people in Deep Security uniforms.`
+			`	As you approach them, the man stops talking with someone and notices you. "Captain <last>! Have you located the Star Queen?"`
+			choice
+				`	"Yes, it is located on <destination>, but is heavily defended."`
+			
+			branch destroyed
+				has "Bactrian Unlock [0]: Pirates: done"
+			`	You mention the multiple heavy warships that you saw over the planet, and the fact that there are probably many more waiting on the planet.`
```

Comment:
"You mention the multiple heavy warships that you saw over the planet" something seems off, I think the "that you saw over the planet" part is